pyproject.toml

```diff
@@ -79,7 +79,7 @@ requires = ["hatchling"]
 build-backend = "hatchling.build"
 
 [tool.hatch.build.targets.wheel]
-packages = ["deploy/sdk/src/dynamo", "components/planner/src/dynamo", "components/ingress/src/dynamo", "components/backends/llama_cpp/src/dynamo"]
+packages = ["deploy/sdk/src/dynamo", "components/planner/src/dynamo", "components/frontend/src/dynamo", "components/backends/llama_cpp/src/dynamo"]
```

Comment:
Given this change messing with the precommits notion of correct import order, I wonder if this is subtlely breaking the relative import of components.frontend - CC @indrajit96 @krishung5 in case this breaks anything, though I'm not sure of the current state of the multimodal example with the refactoring away from dynamo serve and such

Reply:
Yeah by no means a blocking comment, just an FYI that crossed my mind for the image and video enjoyers. Thanks for being on top of it 🚀